dockerfiles/dev/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.5
+COPY requirements.txt /requirements.txt
+RUN python -m pip install -U pip
+RUN python -m pip install -r requirements.txt
+RUN apt-get update && apt-get install -y postgresql postgresql-contrib
+WORKDIR /code
+RUN echo "America/Sao_Paulo" > /etc/timezone && dpkg-reconfigure -f noninteractive tzdata
+VOLUME /code/staticfiles
+CMD ["gunicorn", "jarbas.wsgi:application", "--reload", "--bind", "0.0.0.0:8001", "--workers", "4"]
```

Comment:
We don't need to use a different Dockerfile to prod and dev. Should be the same.